scripts/icm-injection-scripts/test-inject-icm.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+SCRIPTDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null 2>&1 && pwd)"
+
+: "${TEST_SBOM_FORMAT?must set variable; valid values: cyclonedx, spdx}"
+: "${TEST_IMAGE=icm-inject-test:latest}"
+
+banner() {
+    echo "--------------------------------------------------------------------------------"
+    echo "$1"
+    echo "--------------------------------------------------------------------------------"
+}
+
+cleanup() {
+    rm -r "$WORKDIR" || true
+    buildah rm "$test_container" >/dev/null || true
+    buildah rmi "$TEST_IMAGE" >/dev/null || true
+}
+trap cleanup EXIT
+
+WORKDIR=$(mktemp -d --suffix=-icm-inject-test)
+
+banner "Creating test image: $TEST_IMAGE"
+test_container=$(buildah from registry.fedoraproject.org/fedora-minimal:41)
+buildah commit "$test_container" "$TEST_IMAGE"
+
+cd "$WORKDIR"
+
+banner "Running inject-icm.sh with a $TEST_SBOM_FORMAT SBOM"
+cp "$SCRIPTDIR/test-data/sbom-cachi2-$TEST_SBOM_FORMAT.json" ./sbom-cachi2.json
+bash "$SCRIPTDIR/inject-icm.sh" "$TEST_IMAGE" "/tmp"
```

Comment:
Nitpick: you may have copied this script from https://github.com/konflux-ci/build-tasks-dockerfiles/pull/275 rather than main, the `/tmp` argument isn't there in main

Reply:
Fixed